solr/core/src/java/org/apache/solr/handler/admin/LukeRequestHandler.java

```diff
@@ -341,10 +342,9 @@ private static SimpleOrderedMap<Object> getDocumentFieldsInfo(
                     .array(),
                 StandardCharsets.ISO_8859_1));
       }
-      if (!ftype.isPointField()) {
+      if (ftype != null && !ftype.isPointField()) {
         Term t =
-            new Term(
-                field.name(), ftype != null ? ftype.storedToIndexed(field) : field.stringValue());
```

Comment:
ftype is always nonnull - otherwise `ftype.isPointField` would blow up. This simplifies the `new Term` call.